### PR TITLE
Allow nephio home to be configured on install

### PR DESCRIPTION
**Pull request by liamfallon**
The default value of `/home/NEPHIO_HOME`  at line 31 in `init.sh` will never be set because `HOME` is a system variable and always has a value:

```
HOME=${HOME:-/home/$NEPHIO_USER}
```

`init.sh` is amended to pass the `NEPHIO_HOME` variable to the script and change line 31 to

```
HOME=${NEOHIO_HOME:-/home/$NEPHIO_USER}
```

This allows a user to change `HOME` if they like (by setting NEPHIO_HOME) but its likely the default value of `/home/$NEPHIO_USER` will be used in most cases.

### CODE REVIEW

**Review by electrocucaracha — COMMENTED** [review]
The [e2e error](http://prow.nephio.io/view/gs/prow-nephio-sig-release/pr-logs/pull/nephio-project_test-infra/116/e2e/1671491690357067776#1:build-log.txt%3A51515) is not caused by this change. There is a timeout issue that has to be fixed

File: e2e/provision/README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ The following environment variables can be used to configure the installation:
 | ---------------------- | ---------------- | ------------- | ------------------------------------------------------ |
 | NEPHIO_USER            | userid           | ubuntu        | The user to install the sandbox on (must have sudo passwordless permissions) |
 | NEPHIO_DEBUG           | false or true    | false         | Controls debug output from the install                 |
+| NEPHIO_HOME            | a directory      | /home/$NEPHIO_USER | The directory to check out the install scripts into |
```
[the remark this answers is not in the document]
Comment (liamfallon) [reply e2e/provision/README.md]: Done

**Pull request merged**